Update some API changes: (#3759)
timescale/docs: api/refresh_continuous_aggregate.md

changes that only occurred in the secondary table used in the JOIN.
</Highlight>

### Optional arguments

|Name|Type|Description|
|-|-|-|

> | `force` | BOOLEAN | Force the refresh even if the window range is already refreshed. Default to `FALSE`.|

Reviewer: Might be good to emphasize that it is an expensive operation, e.g., something like this:
```suggestion
| `force` | BOOLEAN | Force refresh of every bucket in the time range even if the bucket is already refreshed. Can be very expensive if a lot of data is refreshed. Default to `FALSE`.|
```

> | `force` | BOOLEAN | Force refresh of every bucket in the time range even if the bucket is already refreshed. Can be very expensive if a lot of data is refreshed. Default to `FALSE`.|

Reviewer 2: ```suggestion
| `force` | BOOLEAN | Force refresh every bucket in the time range between `window_start` and `window_end`, even when the bucket has already been refreshed. This can be very expensive when a lot of data is refreshed. Default is `FALSE`.|
```

$$;
```

> Forcing the refresh of the continuous aggregate `conditions` between `2020-01-01` and

Reviewer 2: ```suggestion
Force the  `conditions` continuous aggregate to refresh between `2020-01-01` and
```

> `2020-02-01` exclusive even if it is already refreshed.

Reviewer 2: ```suggestion
`2020-02-01` exclusive, even if the data has already been refreshed.
```

```sql

> CALL refresh_continuous_aggregate('conditions', '2020-01-01', '2020-02-01', TRUE);

Reviewer: Probably good to encourage named parameters to avoid problems in the future.
```suggestion
CALL refresh_continuous_aggregate('conditions', '2020-01-01', '2020-02-01', force => TRUE);
```
